commands/utility/reminder.js

```diff
@@ -29,18 +30,16 @@ class Reminder extends Command {
       reminderTimestamp: message.createdTimestamp + ms(blah.split("#")[1])
     });
 
-    message.channel.send(`I will remind you to \`${blah.split("#")[0]}\`, ${blah.split("#")[1]} from now.`);
+    return message.channel.send(`I will remind you to \`${blah.split("#")[0]}\`, ${blah.split("#")[1]} from now.`);
   }
 
   regCheck(reminder) {
     const remind = /(?:^| )(?:in ?)?(((?:\d{1,2}(?:\.\d|\d)?)|a) ?((?:m(?:in(?:ute)?)?|h(?:our)?|d(?:ay)?|w(?:eek)?|m(?:onth)?|y(?:ear)?)s?))\b/gi.exec(reminder);
     if (!remind) return false;
-    const time = remind[1]
-      .replace(/ ms?\b/, " min") //m => min
+    const time = remind[1].replace(/ ms?\b/, " min") //m => min
       .replace(/\ba ?((?:m(?:in(?:ute)?)?|h(?:our)?|d(?:ay)?|w(?:eek)?|m(?:onth)?|y(?:ear)?)s?)\b/g, "1 $1").trim(); // a "something" => 1 "something"
-    const input = /(?:me ?)?(?:to ?)?(.*)/gi.exec(reminder)[1]
-      .replace(remind[0], "").trim();
-    if (input.length === 0) return false;
+    const input = /(?:me ?)?(?:to ?)?(.*)/gi.exec(reminder)[1].replace(remind[0], "").trim();
+    if (!input.length) return false;
     return `${input}#${time}`;
```

Comment:
Should return an array. What if the user decides to put a `#`? Exactly. It breaks.